src/docs/asciidoc/api-guide/overview/terms-of-service.adoc

```diff
@@ -0,0 +1,18 @@
+[[terms-of-service]]
+== Terms of service
+
+=== Playback information
+As an API user, you are responsible for submitting video playback information to Boclips via our events API. If you are using Boclips' javascript player this will be done automatically, however, if you are not, you must submit these playback events yourselves. More information about how to do this can be found <<resources-events,`here`>>.
```

Comment:
```suggestion
As an API user, you are responsible for submitting video playback information to Boclips via our events API. If you are using <<https://www.npmjs.com/package/boclips-player, Boclips' JavaScript player>> this will be done automatically, however, if you are not, you must submit these playback events yourselves. More information about how to do this can be found <<resources-events,`here`>>.
```

Reply:
Not sure about the link format... I think it would be nice to link to the player for reader's convenience.